pipelines/SConscript

```diff
@@ -0,0 +1,24 @@
+from lsst.sconsUtils.state import env, targets
+from lsst.sconsUtils.utils import libraryLoaderEnvironment
+
+targets.setdefault(
+    "pipelines",
+    []).extend(
```

Comment:
This is pretty unusual formatting; I'd put a newline between `]` and `)` with no indent, but that's mostly just a guess at what black would do.

Reply:
this is what black outputted for me